<?xml version="1.0"?>
<launch>
<!--
    rosrun image_view image_saver __name:=image_saver _save_all_image:=false  image:=/tello/image_raw
    rosrun image_view image_saver __name:=image_saver _save_all_image:=false  image:=/camera6/color/image_raw
 -->
    <node pkg="nodelet" type="nodelet" name="controller" args="manager" required="true" output="screen"/>
    <node pkg="nodelet" type="nodelet" name="track_target"
          args="load tello_controller/track_target_points controller"
          output="screen">

        <param name="queue_size" type="int" value="5"/>
        <param name="start" type="int" value="0"/>
        <param name="offset_x" type="double" value="0"/>
        <param name="offset_y" type="double" value="-0.04"/>
        <param name="offset_z" type="double" value="0.075"/>

        <!--                                     degree -->
        <param name="MAX_ANGULAR" type="double" value="10.0"/>
        <param name="MAX_SPEED" type="double" value="0.1"/>
        <param name="ReachDist" type="double" value=" 0.12"/>
        <!--                                     degree -->
        <param name="ReachAngle" type="double" value="3.0"/>

        <param name="telloFrame" type="string" value="telloFrame0"/>
        <param name="wpFrame" type="string" value="camera7_color_optical_frame"/>
        <param name="enable_wp_loop" type="bool" value="false"/>
        <param name="way_point_file" type="string" value="$(find tello_controller)/way_point/20200518/path_minitype.txt"/>

        <!-- cmd_vel settings -->
        <remap from="cmd_vel" to="/tello/cmd_vel"/>
    </node>
</launch>
